src/Sulu/Bundle/MediaBundle/Resources/config/doctrine/FileVersion.orm.xml

```diff
@@ -35,7 +35,7 @@
 
         <one-to-one field="defaultMeta" target-entity="Sulu\Bundle\MediaBundle\Entity\FileVersionMeta">
             <cascade>
-                <cascade-all />
+                <cascade-persist />
             </cascade>
 
             <join-column name="idFileVersionsMetaDefault" referenced-column-name="id" nullable="true" on-delete="CASCADE" />
```

Comment:
```suggestion
            <join-column name="idFileVersionsMetaDefault" referenced-column-name="id" nullable="true" />
```

We want to remove this CASCADE so when somebody tries to remove the `defaultMeta` without setting a new default meta to the `FileVersion` they should get a SQL error.

Reply:
We are not able todo that because of: https://github.com/sulu/sulu/pull/7282#discussion_r1508878235